File: Örnek-6.3/Java/ucgenalan.java
package algoritma;
import java.util.Scanner;

public class ucgenalan {

	public static void main(String[] args) {
		float a,b,açı,alan;
		float pi=(float)3.14;
		Scanner girdi =new Scanner(System.in);
		
		System.out.print("Birinci kenar uzunluğunu griniz (cm): ");
		a=girdi.nextFloat();
		System.out.print("İkinci kenar uzunluğunu griniz (cm): ");
		b=girdi.nextFloat();
		System.out.print("Arasındaki açıyı giriniz (derece): ");
		açı=girdi.nextFloat();

		alan=(float)(a*b*Math.sin(Math.toRadians(açı))/2);
		System.out.println("Üçgenin alanı (cm2):"+alan);
	}

}

File: Örnek-6.3/Python/ucgenalan.py
import math

a=float(input('Bir kenar uzunluğunu giriniz (cm): '))
b=float(input('İkinci kenar uzunluğunu giriniz: (cm): '))
açı=float(input('Arasındaki açıyı giriniz (derece): '))

alan=a*b*math.sin(math.radians(açı))/2
print('Üçgenin alanı (cm2): %0.3f'%(alan))